scripts/gulp/vendor-bundles.js

```diff
@@ -21,7 +20,6 @@ module.exports = {
     ],
     katex: ['katex'],
     showdown: ['showdown'],
-    unorm: ['unorm'],
```

Comment:
We no longer create the "unorm" bundle because the code goes into the bundle that polyfills String.prototype.normalize.